Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -542,7 +545,12 @@ class CombinatorialKalmanFilter {
                 if (!res.ok()) {
                   ACTS_ERROR("Error in finalize: " << res.error());
                   result.result = res.error();
-                  return;
+                } else {
+                  auto fittedState = *res;
```

Comment:
So when `res.ok() == true` we do this twice?

Reply:
oh - that is a copy paste mistake